src/equations/compressible_mhd_3d.jl

```diff
@@ -0,0 +1,185 @@
+@doc raw"""
```

Comment:
Also, I just realized looking at this file again (as well as both Navier-Stokes files) that it is missing the `@muladd` header
```julia
# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
# Since these FMAs can increase the performance of many numerical algorithms,
# we need to opt-in explicitly.
# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
@muladd begin
#! format: noindent

...

end # @muladd
```
This should be added as well.

Reply:
I changed the name of the files and equations. Still need to check that I did it everywhere I should have.